src/main/java/com/gluonhq/richtext/viewmodel/ActionCmd.java

```diff
@@ -0,0 +1,11 @@
+package com.gluonhq.richtext.viewmodel;
+
+import javafx.beans.binding.BooleanBinding;
+
+public interface ActionCmd {
+    void apply(RichTextAreaViewModel viewModel);
```

Comment:
Would `execute` be a better name for a command?

Reply:
Yes, changed